docker-compose.yaml

```diff
@@ -87,3 +87,7 @@ services:
         build:
             dockerfile: ./docker/slicer/Dockerfile
             context: .
+    generator:
+        build:
+            dockerfile: ./docker/generator/Dockerfile
+            context: .
```

Comment:
It's better not to include generator here, for it has no purpose. Better to use Docker directly in Makefile